Library/Homebrew/formula.rb

```diff
@@ -440,8 +462,10 @@ def bottle_for_tag(tag = nil)
   sig { void }
   def update_head_version
     return unless head?
-    return unless head.downloader.is_a?(VCSDownloadStrategy)
-    return unless head.downloader.cached_location.exist?
+
+    head_spec = T.must(head)
```

Comment:
No way to get sorbet to recognise the `head?` check above to avoid requiring `T.must`?

Reply:
`head` would also be insufficient, as sorbet doesn't assume that a successive invocations of a nilable function (or even an attr) will have the same type